File: Ex45_CountingSpaces.java
import java.util.Scanner;

public class Ex45_CountingSpaces{
	public static void main(String[] args){
		
		int num_spaces = 0;
		
		Scanner input = new Scanner(System.in);
		
		System.out.println("Enter a line ");
		String s = input.nextLine();
		
		for(int i = 0; i<s.length(); i++)
			if (s.charAt(i) == ' ') num_spaces++;
		System.out.printf("Number of spaces: " + num_spaces);
		
	}
}

File: 4th week math char string/Ex45_CountingSpaces.c
#include <stdio.h>
#include <math.h>

int main(){
	
	char str[60];
	int i = 0; int j = 0;
	int num_spaces = 0;
	
	printf("Enter a line: ");
	fgets(str, 60, stdin);
	
	for(j = 0; str[j]!='\0';++j);
	
	for(i = 0; i<j; i++){
		if(str[i] == ' '){
			num_spaces++;
		}
	}
	
	printf("Number of spaces: %d", num_spaces);	
	
	return 0;
}
